packages/insomnia-app/app/ui/components/editors/environment-editor.js

```diff
@@ -54,7 +54,7 @@ class EnvironmentEditor extends React.PureComponent<Props, State> {
 
     // Check for invalid key names
     if (value) {
```

Comment:
Looks like this condition should also be updated

```suggestion
    if (value && value.object) {
```

Reply:
done